examples/swr-site/pages/docs/data-fetching.en-US.mdx

```diff
@@ -13,7 +12,7 @@ The returned value will be passed as `data`, and if it throws, it will be caught
 
 <Callout emoji="💡">
   Note that <code>fetcher</code> can be omitted from the parameters if it's{' '}
-  <Link href="/docs/global-configuration"><a href="/docs/global-configuration">provided globally</a></Link>.
+  [provided globally](/docs/global-configuration).
```

Comment:
there was no reason to use Link in this place (also in current swr https://github.com/vercel/swr-site/blob/main/pages/docs/getting-started.en-US.mdx)
Because `components.a` was configured to use either `<a/>` tag for `https://` links or `<NextLink />` for internal routes :)